src/js/components/homepage/hero/AnimatedHeading.jsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Hero.jsx
+ * CardHero.jsx
```

Comment:
I don't think we need to rename this file here.
Was this a result of a find and replace?